### PR TITLE
Use newer rollup for better tree-shaking

### PR DESCRIPTION
**Pull request by skeggse**

### CODE REVIEW

**Review by ttacon — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,18 @@
 ## Release History
 
+* 3.0.0 Use latest rollup by using our own rollup-stream
```
Comment (wearhere) [inline CHANGELOG.md:3]: Oh neat!!

File: index.js [hunk 2]
```diff
@@ -118,27 +120,27 @@ class MultiBuild {
...
 
         return options.output(
-            target,
-            rollup(rollupOptions)
-              .on('error', function(e) {
-                if (options.errorHandler) {
-                  this.emit('end');
-                  options.errorHandler(e);
-                } else {
-                  throw(e);
```
Comment (wearhere) [inline index.js:128]: wat. haha

File: index.js [hunk 3]
```diff
@@ -24,16 +26,16 @@ class MultiBuild {
    *    Must not contain spaces since they will be used to form the names of the build tasks.
    *  @param {Array<String>} [skipCache] - Names of targets that should not use rollup's cache, eg
    *  because they are processed differently than other targets. Defaults to [].
-   *  @param {Function} entry - A function that returns the Rollup entry point when invoked with a
+   *  @param {Function} input - A function that returns the Rollup entry point when invoked with a
```
Comment (wearhere) [inline index.js:29]: nit: I can see how this is more consistent with `output` but if you didn't change this would this diff still be breaking?

**Review by wearhere — COMMENTED** [review]
yay glad to improve tree-shaking!

File: CHANGELOG.md (hunk 1 above)
Comment (wearhere) [inline CHANGELOG.md:3]: How is this a breaking change? Can't tell from the rest of the diff

File: index.js (hunk 3 above)
Comment (wearhere) [reply index.js:29]: clarified with update to changelog above

**Pull request merged**